lib/swoosh/email.ex

```diff
@@ -464,23 +464,30 @@ defmodule Swoosh.Email do
   as an argument. If you give a path we will detect the MIME type and determine the filename
   automatically.
 
+  You can also send an inline-attachment used for embedding images in the body of emails by specifying `type: "inline"`
```

Comment:
Currently it's `:inline` not `"inline"`
(might change, see comment above)

Reply:
👍